### ISSUE TITLE
想要修改tensor的值，但不希望被记录。有没有类似tensor.data获取方式啊?

### ISSUE THREAD

**Issue by MARKLIN1000** [PaddlePaddle/Paddle#43593]
### 请提出你的问题 Please ask your question

![72999d9623ed5e03ddda489b6b25d2c](https://user-images.githubusercontent.com/82072274/174061540-bd402a7b-90eb-4c29-bf65-25756ad1331b.png)

想要修改 tensor的数值，但是又不希望被autograd记录，有没有ptorch中 tensor.data 的替代API

**Comment by an1018** [comment]
可以参考这个
<img width="298" alt="45b38d95a22158f0c573a261ea10a1cf" src="https://user-images.githubusercontent.com/19808900/174211202-4dfbb3c5-35bb-4575-8a9f-4402d13cf6d8.png">

**Comment by hua-zi** [comment]
> 可以参考这个 <img alt="45b38d95a22158f0c573a261ea10a1cf" width="298" src="https://user-images.githubusercontent.com/19808900/174211202-4dfbb3c5-35bb-4575-8a9f-4402d13cf6d8.png">

请问使用 `tensor.set_value()` 或者 `tensor.detach()` , 可以不被autograd记录吗？
`tensor.detach()` 是否可以完全取代 `tensor.data` 呀
